src/lib.rs

```diff
@@ -2,5 +2,6 @@ mod error;
 mod rate_limiter;
 mod token_bucket;
 
-pub use rate_limiter::RateLimiter;
+pub use error::Error;
+pub use rate_limiter::{RateLimiter, RateLimiterBuilder};
```

Comment:
I wonder if we even want to export `RateLimiterBuilder` at this level. As in, I believe the intended usage pattern is to construct a `RateLimiterBuilder` by calling `RateLimiter::new()`?

Reply:
Ah, right. We either need to make the `mod` above `pub mod`, or to re-export the name using `pub use` like we do now. Let's keep it!